### DISCUSSION TITLE
fail to test doris on hive

### DISCUSSION THREAD

**Discussion by young138120** [apache/doris#8845] [A - General / Q&A]
i according to the [doc ](https://doris.apache.org/zh-CN/extending-doris/hive-of-doris.html#%E5%90%8D%E8%AF%8D%E8%A7%A3%E9%87%8A) steps to test ,but its failed

i get the error messge : 
**ERROR 1064 (HY000): errCode = 2, detailMessage = Query from hive table is not supported, table: test_1**

create hive table sql:
```sql
CREATE EXTERNAL TABLE `test_1` (
  `test_id` BIGINT COMMENT 'id',
  `name` STRING COMMENT 'name')
PARTITIONED BY (day STRING COMMENT 'yyyyMMdd')  
STORED AS orc 
LOCATION 'hdfs:/path/to/save/test'
TBLPROPERTIES (
  'orc.compress' = 'snappy');
```

create doris table sql:
```sql
CREATE EXTERNAL TABLE `test_1` (
  `test_id` BIGINT COMMENT 'id',
  `name` VARCHAR COMMENT 'name',
  `day` VARCHAR COMMENT 'fenqu'
) ENGINE=HIVE
COMMENT "HIVE"
PROPERTIES (
'hive.metastore.uris' = 'thrift://x.x.x.x:9083',
'database' = 'test',
'table' = 'test_1'
);
```

i am not found any message from google by the key word which is doris Query from hive table is not supported

its really not supported?

**Comment by hf200012** [comment]
What is your doris version? Support hive appearance query need doris 1.0 version to be able to, the previous version does not support

**Reply by young138120** [reply]
sure？
I used the 0.15.0 version , it is latest release version on now。

https://doris.apache.org/installing/compilation.html#developing-mirror-compilation-using-docker-recommended

**Reply by young138120** [reply]
and how can i upgrade version

**Reply by Jibing-Li** [reply]
Here is the link to the latest release version, which supports hive external table. 
https://github.com/apache/incubator-doris/tree/1.0.0-rc03

**Reply by Jibing-Li** [reply]
To upgrade version, I suggest back up your 0.15.0 cluster first, so that you can restore your cluster if anything goes wrong during the upgrading. 
Then, compile the 1.0.0 release and replace the old lib directories of both be and fe with the new 1.0.0 lib.
Restart your cluster.

**Reply by young138120** [reply]
> Here is the link to the latest release version, which supports hive external table. https://github.com/apache/incubator-doris/tree/1.0.0-rc03
3Q， i will try it

**Reply by young138120** [reply]
> To upgrade version, I suggest back up your 0.15.0 cluster first, so that you can restore your cluster if anything goes wrong during the upgrading. Then, compile the 1.0.0 release and replace the old lib directories of both be and fe with the new 1.0.0 lib. Restart your cluster.

I'll close the question after I try it 
i hope i will be successful

**Reply by hf200012** [reply]
> sure？ I used the 0.15.0 version , it is latest release version on now。
> 
> https://doris.apache.org/installing/compilation.html#developing-mirror-compilation-using-docker-recommended

This is only supported in version 1.0
